.changeset/sweet-hairs-remember.md

```diff
@@ -0,0 +1,5 @@
+---
+'@qwik.dev/core': major
+---
+
+FIX: Add a new e2e test to verify QRL behavior without uncaught promises. Introduce retry logic for QRL resolution to handle potential promise retries, ensuring robustness in asynchronous operations.
```

Comment:
this message will appear on patch notes, so it should contain information for developers. I think information about e2e test is not useful :D I would leave only the second sentence